### PR TITLE
Feature  rand centr infmax

### PR DESCRIPTION
**Pull request by anty-filidor**
This PR brings new seed selection classses based on non-ml approaches: random choice, centralities and ground truth.

Merge available only after closure of https://github.com/network-science-lab/ns-data-sources/pull/15 and updating a reference pointing to `ns-data-sources`

### CODE REVIEW

**Comment by anty-filidor** [comment]
@mateuszStolarski - please review as well

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Then, pull the submodule with data loaders and install its code:
```
Comment (mateuszStolarski) [inline README.md:19]: please update branch and resolve conflicts

File: _configs/eval_ssm.yaml [hunk 2]
```diff
@@ -2,56 +2,82 @@ spreading_model:
   name: "TorchMICModel"
   parameters:
     protocols: ["OR"]
-    p_values: [0.25, 0.5, 0.75]
+    p_values: [0.05, 0.10, 0.15, 0.20]
```
Comment (mateuszStolarski) [inline _configs/eval_ssm.yaml:5]: (2) this name 'p_values' is confusing, may be thresholds will be more suitable

Comment (anty-filidor) [reply _configs/eval_ssm.yaml:5]: but we're using unfortunately this term everywhere. that's why i decided to keep it here

Comment (mateuszStolarski) [reply _configs/eval_ssm.yaml:5]: please leave todo for that, I'll change that in free time as low priority task

Comment (anty-filidor) [reply _configs/eval_ssm.yaml:5]: done

File: README.md (hunk 1 above)
Comment (anty-filidor) [reply README.md]: done

**Pull request merged**